web/beacon-app/src/index.css

```diff
@@ -157,6 +157,18 @@
   min-height: calc(100vh - 600px);
 }
 
+.create-key-modal, .download-key-modal {
```

Comment:
This style change instructs the browser to choose either a minimum width of `50vw` or `600px` for screens with a minimum width of 640px. `!important` is included to override styles set in the Beacon design system for all modals.